From 50a89fc53b5a66d6478842bdf1ad87852307139d Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:51:32 -0400
Subject: [fs] nfsd: remove unneeded linux/user_namespace.h include

Message-id: <1403560314-32525-151-git-send-email-bfields@redhat.com>
Patchwork-id: 84539
O-Subject: [Patch RHEL7 150/172] NFSD: remove unneeded linux/user_namespace.h include
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Kinglong Mee <kinglongmee@gmail.com>

After commit 4c1e1b34d5c8 ("nfsd: Store ex_anon_uid and ex_anon_gid as
kuids and kgids") using kuid/kgid for ex_anon_uid/ex_anon_gid,
user_namespace.h is not needed.

Signed-off-by: Kinglong Mee <kinglongmee@gmail.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 0faed901c680b153ca090a766ceda699b55993eb
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/auth.c b/fs/nfsd/auth.c
index 06cddd5..d69dd9a 100644
--- a/fs/nfsd/auth.c
+++ b/fs/nfsd/auth.c
@@ -1,7 +1,6 @@
 /* Copyright (C) 1995, 1996 Olaf Kirch <okir@monad.swb.de> */
 
 #include <linux/sched.h>
-#include <linux/user_namespace.h>
 #include "nfsd.h"
 #include "auth.h"
 
-- 
1.7.1